code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -4,7 +4,12 @@
 	id = SPECIES_LIZARD
 	say_mod = "hisses"
 	default_color = "00FF00"
-	species_traits = list(MUTCOLORS, EYECOLOR, LIPS, HAS_FLESH, HAS_BONE, TRAIT_CAN_USE_FLIGHTPOTION)
+	species_traits = list(MUTCOLORS, EYECOLOR, LIPS, HAS_FLESH, HAS_BONE)
+	inherent_traits = (
+		TRAIT_ADVANCEDTOOLUSER,
+		TRAIT_CAN_STRIP,
+		TRAIT_CAN_USE_FLIGHT_POTION
```

Comment:
trailing comma

Reply:
look you get it, do it everywhere

Reply:
yeah i think i fixed it in the "oops" commit but thanks for the other feedback :)